src/app/(site)/create/page.tsx

```diff
@@ -133,16 +159,24 @@ export default function CreateBounty() {
     const onSuccess = (_: Event) => {
       void invalidateKeys(["open", "posted"]);
       if (shouldNotify) {
-        postSocialNote();
+        postSocialNote(_)
```

Comment:
This is a promise, but maybe we should update react-nostr to be able to accept an async onSuccess function (line 159)